.travis.yml

```diff
@@ -56,24 +59,30 @@ env:
   - secure: KX7DwKRD85S7NgspxevgbulTtV+jHQIiM6NBus2/Ur/P0RMdpt0EQQ2wDq79qGN70bvvkw901N7EjSYd+GWCAM7StXtaxnLRrrZ3XI1gX7KMk8E3QzPf0zualLDs7cuQmL6l6WiElUAEqumLc7WGpLZZLdSPzNqFSg+CBKCmTI8=
 
 git:
-  depth: 60
+  depth: 50
 
 before_install:
   - travis/before_install.sh
 
 install:
-  - source travis/install-conda.sh
+  - if [[ "$BUILD_PACKAGE" != "docker" ]]; then source travis/install-conda.sh; fi
   - if [ -n "$PERFORM_TESTING" ]; then source travis/install-tools.sh; fi
   - if [ -n "$PERFORM_TESTING" ]; then travis/install-pip.sh; fi
+  - if [[ "$BUILD_PACKAGE" == "conda" ]]; then
+      conda install -q -y conda-build==3.0.25 &&
+      conda install -q -y anaconda-client &&
+      conda create -m -q -y -p tools/conda-tools/default Jinja2==2.8;
```

Comment:
Extra semicolon the result of a one-liner that grew?

Reply:
Oops.. yeah. Could be separate script too. I wonder if I can `set -e` in a travis.yml line? Otherwise I might want to replace the semicolon with a `|| exit 1`.